easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.04-Apollo-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,93 @@
+        'filename': 'xmippCore-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippViz/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippViz-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/DStrelak/cuFFTAdvisor/archive/'],
+        'download_filename': '14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz',
+        'filename': 'cuFFTAdvisor-20191010.tar.gz',
+    },
+]
+patches = [
+    '%(name)s-%(version)s_fix_scons_Environent_calls.patch',
+    '%(name)s-%(version)s_use_CXXFLAGS_from_EB.patch',
+    '%(name)s-%(version)s_add_missing_pthread_to_XmippParallel.patch',
+    '%(name)s-%(version)s_add_missing_pthread_to_XmippInterfaceCuda.patch',
+    '%(name)s-%(version)s_dont_add_useless_libs.patch',
+    '%(name)s-%(version)s_correct_order_of_cu_includes.patch',
+    'XmippCore-%(version)s_fix_scons_Environent_calls.patch',
+    'XmippCore-%(version)s_use_CXXFLAGS_from_EB.patch',
+    'XmippViz-%(version)s_fix_scons_Environent_calls.patch',
+    'XmippViz-%(version)s_use_CXXFLAGS_from_EB.patch',
+]
+checksums = [
+    '1fcad53aacafca5c1ecf4350b83679178c40e8fd1984282a9e2359e2ae5b84e0',  # xmipp-v3.19.04-Apollo.tar.gz
+    '0f540db0e8e264443974ba761d447b7c30bfdd6f63258a010e4875edcf658487',  # xmippCore-v3.19.04-Apollo.tar.gz
+    '49a4180f3f9a07208b78715a3a6a47d02d0537008d4c25d9812c1574f14f2676',  # xmippViz-v3.19.04-Apollo.tar.gz
+    '2d97cc1c9c955fa5bee44eb232d49f25f2c8ad164ea2be754d44c723c613be46',  # cuFFTAdvisor-20191010.tar.gz
+    # Xmipp-3.19.04-Apollo_fix_scons_Environent_calls.patch
+    '8cab4ed9ba1a4a5aa36e01de8d25e726371c35a5271ac75fbabc8bec08344397',
+    # Xmipp-3.19.04-Apollo_use_CXXFLAGS_from_EB.patch
+    '839b602098d16c4ba8808b1e826a66262101dde3a65d21c7956f3d22678625b3',
+    # Xmipp-3.19.04-Apollo_add_missing_pthread_to_XmippParallel.patch
+    '5b68f16197b460a2dd734131701fdec8ce0b226fef3e523b70aa369e7874df53',
+    # Xmipp-3.19.04-Apollo_add_missing_pthread_to_XmippInterfaceCuda.patch
+    'afd848a3a3bb7d225d80fab6060ac80eb1093b966e90239c7631c45718bb28cc',
+    # Xmipp-3.19.04-Apollo_dont_add_useless_libs.patch
+    '6afea998734306e1eb02010b86b379ee7c589eef4882dfb611a29189295b0bc5',
+    # Xmipp-3.19.04-Apollo_correct_order_of_cu_includes.patch
+    'ec746799dbb9da500cb02c4eb7b6cd9089eeec31a69259e4966ac25be16f0cdf',
+    # XmippCore-3.19.04-Apollo_fix_scons_Environent_calls.patch
+    '999794c74e1589452debee091f151b76c98bbc806fa0f16e7713b5741e0068c9',
+    # XmippCore-3.19.04-Apollo_use_CXXFLAGS_from_EB.patch
+    'da4bfd9aa9fc90aa8168c1e62c9398da7fd0ca22f2aee5fd890c35cae2ae0edb',
+    # XmippViz-3.19.04-Apollo_fix_scons_Environent_calls.patch
+    '94aedf2c3728eb53ae1b2473457ce4f9a6fc0912c90a3593ab51e85fc308bc4d',
+    # XmippViz-3.19.04-Apollo_use_CXXFLAGS_from_EB.patch
+    '18a6ea99eb4deae60e55fff1052980f2ef377a7fd74cfdba2af922e25f76e3d3',
+]
+
+builddependencies = [
+    ('SCons', '3.0.5'),
```

Comment:
@akesandgren Test build fails because the `Python/3.7.2` that is loaded by default by `SCons` clashes with the Python 7.1.5 that is listed as a dependency.

Workaround for now (until we come up with a better solution, if we can), is to also list `Python` 2.7.15 as a build dep, *before* SCons (which makes it use Python 2 rather than loading Python 3.7.2 which it does by default):

```python
builddependencies = [
    ('Python', '2.7.15'),
    ('SCons', '3.0.5'),
]